recipe/recipe.yaml

```diff
@@ -1,20 +1,20 @@
 context:
   name: k9s
-  version: 0.40.5
+  version: 0.50.4
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
   - url: https://github.com/derailed/${{ name }}/archive/refs/tags/v${{ version }}.tar.gz
-    sha256: e8f886bd95de67b8239b5444776604d45530aee8bb779229425855e7e1d37d4e
+    sha256: 4f7f6702145b0eea2f42c684ff823aa261bc80b1a3b2a33b56fb6f87d7755c73
     target_directory: src
   - url: https://raw.githubusercontent.com/xi2/xz/refs/heads/master/LICENSE
     sha256: 5c640056d513f276d56635af7a0bed81cb7eac02a31b6e56d40c416557a8d8ce
     target_directory: xz-LICENSE-1
   - url: https://raw.githubusercontent.com/tukaani-project/xz/refs/heads/master/COPYING
-    sha256: aedbbc280610875b83e246f05957369f84acf9abdb86b884a9ff2f5dd65e8ffb
+    sha256: 616a3ad264ce29b8f1cb97e53037b139d406899ca8d1f799651e17bfa09830b8
```

Comment:
https://github.com/tukaani-project/xz/commit/70f1f203789433b5d7b8b22e1655abc465d659f7 is what changed